guides/release/components/block-content.md

```diff
@@ -22,6 +22,24 @@ This is equivalent to explicitly naming the default block using the named block
 </ExampleComponent>
 ```
 
+Or consider the following modal component which utilizes a content block and a footer block.
+
+```handlebars
+<div class="modal" role="dialog">
+    <button type="button">Close modal</button>
```

Comment:
Can this code block be adjusted to use 2 space indents so it matches the others?